import/index_java/src/org/vufind/index/FormatCalculator.java

```diff
@@ -548,6 +553,55 @@ protected boolean hasSerialHost(Record record) {
         return false;
     }
 
+    protected boolean isOnlineAccordingTo338(Record record) {
+        // Does the RDA carrier indicate that this is online?
+        List carrierFields = record.getVariableFields("338");
+        if (carrierFields != null) {
+            Iterator carrierFieldsIter = carrierFields.iterator();
+            while (carrierFieldsIter.hasNext()) {
```

Comment:
You should be able to save multiple lines of code by using a `for` loop instead:
```
for (VariableField variableField : record.getVariableFields("338")) {
    DataField carrierField = (DataField) variableField;
```